src/main/java/frc/robot/Constants.java

```diff
@@ -52,6 +54,7 @@ public static final class PathConstants {
     public static final double KP = 0.00000094597;
     public static final double KI = 0;
     public static final double KD = 0;
+    public static final double MAX_VISION_TO_ODOMETRY_ERROR = 1;
```

Comment:
What does this mean? Is there a unit for this?

Reply:
whoops, don't need that anymore